ginicapture/src/doc/source/integration.rst

```diff
@@ -0,0 +1,528 @@
+Failing to do so will throw an exception. The listener instance can be provided either implicitly by making the host
+activity implement the ``CameraFragmentListener`` interface or explicitly by setting the listener using ``setListener()``.
+
+See the reference documentation of this fragment's `public interface
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.camera/-camera-fragment-interface/index.html>`_
+and it's `listener
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.camera/-camera-fragment-listener/index.html>`_
+for more details.
+
+GiniCaptureCoordinator
+++++++++++++++++++++++
+
+This coordinator helps to implement the default behavior of the SDK. For example it helps to show the onboarding screen
+at first launch.
+
+See the `reference documentation
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture/-gini-capture-coordinator/index.html?query=public%20class%20GiniCaptureCoordinator>`_
+for details.
+
+OnboardingFragmentCompat
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+This fragment shows the onboarding screen. You can use the ``GiniCaptureCoordinator`` to know when to show it. You can
+also implement a button to allow users to view it on demand.
+
+It displays important advice for correctly photographing a document.
+
+The default way of showing this fragment is as an overlay above the camera screen with a semi-transparent
+background.
+
+By default an empty last page is added to enable the revealing of the camera preview before this fragment is dismissed.
+You can disable this by using the appropriate ``createInstance...()`` factory  method.
+
+If you would like to display a different number of pages, you can use the factory methods and provide a list of
+``OnboardingPage`` objects.
+
+An ``OnboardingFragmentListener`` instance must be available before the fragment is attached to an
+activity. Failing to do so will throw an exception. The listener instance can be provided either implicitly by making
+the host activity implement the ``OnboardingFragmentListener`` interface or explicitly by setting the listener using
+``setListener()``.
+
+See the `reference documentation
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.onboarding/-onboarding-fragment-compat/index.html?query=public%20class%20OnboardingFragmentCompat%20extends%20Fragment%20implements%20OnboardingFragmentImplCallback,%20OnboardingFragmentInterface>`_,
+`public interface
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.onboarding/-onboarding-fragment-interface/index.html>`_
+and `listener
+<http://developer.gini.net/gini-capture-sdk-android/ginicapture/dokka/ginicapture/net.gini.android.capture.onboarding/-onboarding-fragment-listener/index.html>`_
```

Comment:
These are two identical links? It is intended, right?

Reply:
They are not identical: one is `...onboarding-fragment-interface...` and the other is `onboarding-fragment-listener`. 😉